clients/web/src/lib/components/atoms/Button/Button.svelte

```diff
@@ -0,0 +1,7 @@
+<script lang="ts">
+    export let label:string;
+</script>
+
+<button type="button" class="text-white bg-primary-700 hover:bg-secondary-800 
```

Comment:
Needs to handle `on:click`

See https://svelte.dev/repl/a295667e0f384c6da13d58f92a24d934?version=3.55.0

Reply:
Done.